packages/components/src/toolbar/style.scss

```diff
@@ -11,20 +11,33 @@
 }
 
 // @todo: We should extract this to a separate component, and refactor it.
-.block-editor-block-toolbar .components-tab-button {
-	padding: $grid-unit-10 $grid-unit-15;
-	color: $border-color;
-	height: $icon-button-size;
-	font-weight: 500;
+.block-editor-block-toolbar .components-toolbar {
```

Comment:
It bothers me that a style targetting the block-editor-toolbar is written in the generic "toolbar" component. Should we move this style to the BlockToolbar component stylesheet instead? We'll most likely find related styles there too.

Reply:
I just switched branches to look at a different thing. If you feel adventurous, feel free to, otherwise I will have to look in a bit!

Reply:
Looking 👍